spigot_v1_16_R1/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/Spigot_v1_16_R1.java

```diff
@@ -705,14 +706,14 @@ private void regenForWorld(Region region, EditSession editSession, WorldServer s
                 blockEntity.save(tag);
                 state = state.toBaseBlock(((CompoundTag) toNative(tag)));
             }
-            editSession.setBlock(vec, state);
+            extent.setBlock(vec, state.toBaseBlock());
```

Comment:
Why toBaseBlock here?

Reply:
The extent setblock doesn't seem to accept the generic type BlockStateHolder